tests/test_priors.py

```diff
@@ -115,9 +129,59 @@ def testUniform(self):
             m.x = np.random.randn(1)[0]
             p2 = m.compute_log_prior()
 
-            # prior should no be the same because a transformation has been applied.
+            # prior should not be the same because a transformation has been applied.
             self.assertTrue(p1 != p2)
 
 
+def mc_moments(prior, size=(10000, 1000)):
+    np.random.seed(1)
+    x = prior.sample(size)
+    assert x.shape == size
+    mean = x.mean()
+    var = x.var()
+    return mean, var
+
+def gaussian_moments(prior):
+    return prior.mu, prior.var
+
+def exponential_moments(prior):
+    return 1 / prior.rate, prior.rate ** (-2)
+
+def lognormal_moments(prior):
+    mu, var = prior.mu, prior.var
+    return np.exp(mu + var / 2), (np.exp(var) - 1) * np.exp(2 * mu + var)
+
+def gamma_moments(prior):
+    return prior.shape * prior.scale, prior.shape * prior.scale ** 2
+
+def laplace_moments(prior):
+    return prior.mu, 2 * prior.sigma ** 2
+
+def beta_moments(prior):
+    a, b = prior.a, prior.b
+    return a / (a + b), (a * b) / ((a + b)**2 * (a + b + 1))
+
+def uniform_moments(prior):
+    a, b = prior.lower, prior.upper
+    return (a + b) / 2, (b - a)**2 / 12
+
+@pytest.mark.parametrize("args", [
+    ("Exponential", [1.3]),
```

Comment:
All distributions have parameters. You have to set them to *some* value. I didn't want to set them to things like 0 or 1 because that often leads to incorrectly correct-appearing results (due to faulty terms cancelling).

Reply:
That's a point - _some_ values. E.g. tuple (0.1, 0.2) can be a (mu, var) or (var, mu). Can you make it dict(mu=0.1, var=0.2)?

Reply:
But it really doesn't matter *what* they are, whether one is 0.1 and the other 0.2 or vice versa

Reply:
(as long as they obey the constraints on parameters such as variance > 0, and lower < upper for uniform distribution, but that ought to be checked by the `__init__` anyways)